src/components/ScanPage/Trottinette/Trottinette1.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Animated } from "react-animated-css";
+import troot from "../../../Image/lime-bordeaux-1-1024x768.jpg";
+import { Link } from 'react-router-dom';
+
+const Trottinette1 = () => (
+  <Animated animationIn="fadeInUp" animationInDuration='500' animationOut="fadeOut" isVisible={true}>
+    <div className="container_Content">
+      <h1 className="title_troot">Trottinette</h1>
+      <img className='Preview_troot' src={troot} alt="gift" />
```

Comment:
only difference with Trottinette2.jsx is img src: use one component and pass the src as a prop